runners/google-cloud-dataflow-java/pom.xml

```diff
@@ -245,6 +245,7 @@
               -->
               <excludedGroups>
                 org.apache.beam.sdk.testing.LargeKeys$Above10MB,
+                org.apache.beam.sdk.testing.UsesAttemptedMetrics,
```

Comment:
We don't execute streaming tests with this configuration, and when we do we'll have a second configuration. See https://github.com/apache/beam/pull/2947